release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages_System.Drawing.md

```diff
@@ -0,0 +1,76 @@
+# System.Drawing
+
+``` diff
+ namespace System.Drawing {
+-    public static class SystemColors {
+ {
+-        public static Color ActiveBorder { get; }
```

Comment:
This should now be exposed through System.Drawing.Primitives for 3.0, so LGTM.

Reply:
Were they also removed from the standalone package when in-boxed?